EPPlus/Table/PivotTable/ExcelPivotTable.cs

```diff
@@ -858,6 +859,10 @@ internal int CacheID
 		/// Gets or sets the cache definition-pivot table relationship.
 		/// </summary>
 		internal Packaging.ZipPackageRelationship CacheDefinitionRelationship { get; set; }
+
+		private List<PivotTableHeader> RowHeaders { get; set; } = new List<PivotTableHeader>();

```

Comment:
Are you using the set on these properties? If not get rid of it. 
```suggestion
		private List<PivotTableHeader> RowHeaders { get; } = new List<PivotTableHeader>();
```